aws/defaults/handlers.go

```diff
@@ -158,9 +158,10 @@ func handleSendError(r *aws.Request, err error) {
 			Body:       ioutil.NopCloser(bytes.NewReader([]byte{})),
 		}
 	}
-	// Catch all other request errors.
+
+	// Catch all request errors, and let the default retrier determine
```

Comment:
np: reference as the interface type name "Catch all request errors, and let the retryer determine if error is retryable)

Reply:
I didn't get what you meant by this, can you elaborate?